### PR TITLE
Update to_gc_num.mdx

### PR DESCRIPTION
**Pull request by slimerunner**
adding details about invalid GC string

### CODE REVIEW

File: docs/scripting/scripts.lib/to_gc_num.mdx [hunk 1]
```diff
@@ -18,7 +18,14 @@ The formatted GC string to be turned into a number.
 
 ### Return
 
-Returns a number.
+Returns a number or an object (inactive GC string).
```
Comment (tukib) [inline docs/scripting/scripts.lib/to_gc_num.mdx:21]:
```suggestion
Returns a number.

For invalid `gc_string` values, the following object is returned:
```

Separate expected return from exceptional case for extra clarity.

File: docs/scripting/scripts.lib/to_gc_num.mdx [hunk 2]
```diff
@@ -18,7 +18,14 @@ The formatted GC string to be turned into a number.
...
+
+```js
+{
+  ok: false,
+  msg: "GC string " + gc_string + " is not in correct format. Example GC strings: \"1M234K567GC\", \"1B5KGC\", \"2MGC\", \"9GC\"."
```
Comment (tukib) [inline docs/scripting/scripts.lib/to_gc_num.mdx:26]:
```suggestion
  msg: "GC string \"" + gc_string + "\" is not in correct format. Example GC strings: \"1M234K567GC\", \"1B5KGC\", \"2MGC\", \"9GC\"."
```
Missing quotes here

**Review by tukib — COMMENTED** [review]
Thanks for the contribution @slimerunner! I've added a couple suggestions to take a look at, but otherwise this looks good to me

**Review by matr1x-hackmud — APPROVED** [review]

**Comment by matr1x-hackmud** [comment]
Looks good to me. If you're cool with these modifications @slimerunner I'll get this merged in for you :+1:

Alternatively, after a few  days I'll merge it regardless. Thank you for your contribution!

**Pull request merged**